moveit_core/trajectory_processing/src/time_optimal_trajectory_generation.cpp

```diff
@@ -982,9 +982,9 @@ bool TimeOptimalTrajectoryGeneration::computeTimeStamps(
   double velocity_scaling_factor = verifyScalingFactor(max_velocity_scaling_factor, VELOCITY);
   double acceleration_scaling_factor = verifyScalingFactor(max_acceleration_scaling_factor, ACCELERATION);
 
-  const unsigned num_joints = group->getVariableCount();
-  const std::vector<std::string>& vars = group->getVariableNames();
   const moveit::core::RobotModel& rmodel = group->getParentModel();
+  const std::vector<std::string>& vars = group->getActiveVariableNames();
+  const unsigned num_joints = group->getActiveVariableCount();
```

Comment:
@henningkayser this is the change that breaks the unit tests. I think because this just scales all the joints (mimic or not), we probably want to keep it as it was